paf-mvp-cmp/README.md

```diff
@@ -0,0 +1,108 @@
+# OneKey Reference Consent Management Platform (CMP) UI
+
+# Overview
+
+The output from this project is a single JavaScript resource file that contains the entire CMP.
+
+The functionality requires a CMP provider that has enabled CORS access from the publisher or advertiser domain. The host
+name of this CMP provider will be needed before deploying the CMP UI.
+
+# For Publishers and Advertisers
+
+The script adds the HTML elements to the DOM at the point it is included. It should be a blocking include to render the
+UI as soon as possible and enable the web site to continue loading and fetching data whilst the user interacts with the 
+dialogue. All the resources needed to provide the UI are embedded in the single small JavaScript resource.
+
+## Performance (1)
+
+To improve overall performance a preload link element should be added at the beginning of the `<head>` element. This 
+will commence fetching the JavaScript resource as soon as possible in the page render process. This will be 
+particularly noticeable when HTTPS/2 is used.
+
+```html
+<link rel="preload" as="script" href="https://domain.com/assets/cmp/ok-ui.js">
+```
+
+If this link is not added then the functionality will still be provided by page load time will be marginally impaired.
+
+## Script
+
+The script element and attributes must be added to the `<body>` element. See the following example.
+
+```html
+<script src="https://domain.com/assets/cmp/ok-ui.js" 
+    data-display-intro="true"
+    data-snackbar-timeout-ms="5000" 
+    data-proxy-host-name="cmp.pafdemopublisher.com" 
+    data-brand-name="CMP PAF Demo Pub"
+    data-site-only-cookie-tcf-core="pub-tcf"
+    data-brand-privacy-url="https://github.com/prebid/paf-mvp-implementation"></script>
+```
+
+The DOM is modified at the point the script is included. There are no other deployment options.
+
+### Performance (2)
+
+By placing the script element for the CMP as close to the top of the `<body>` element as possible the interface will 
+appear on first request while the rest of the page is loading behind it. Should the interface require a redirect to 
+fetch or store data then the assets for the publishers page will already be cached by the web browser effectively 
+improving user perceive page load time. Also the approach is likely to improve actual page load time compared to 
+solutions which load the interface after other page processing completes.
+
+## Configuration options
+
+The following attributes must be present in the script tag.
+
+- data-brand-name: The brand name to use throughout the user interface.
+- data-brand-privacy-url: This URL is needed to inform the user about the privacy policy of the brand.
```

Comment:
I have the feeling the "brand" is nothing more than what the proxy hosts: I mean, the proxy used here is the one that create seeds that are then used for transmissions.
If this is the case, then I believe brand name and privacy URL should not be a parameter, but something that a call to `<proxyHost>/paf/v1/identity` should provide.